src/scales/scale.radialLinear.js

```diff
@@ -309,12 +309,12 @@ class RadialLinearScale extends LinearScaleBase {
 		const me = this;
 
 		// Set the unconstrained dimension before label rotation
-		me.width = me.maxWidth;
-		me.height = me.maxHeight;
-		me.paddingTop = getTickBackdropHeight(me.options) / 2;
+		me.width = me._maxWidth;
```

Comment:
Don't you think there is a design flaw when a scale needs to access private properties?

Reply:
Hmm. Well my main goal was to make it clear that this property wasn't accessed from outside the scale. I guess that marking it `@protected` might be the most appropriate, though we've not done that anywhere else and I'm not sure whether we'd want to prefix with an underscore or not in that case. I suppose we probably wouldn't

If we want to treat them as private, one option would be to override `update` in the radialLinear scale and store `maxWidth` and `maxHeight` when they're passed in, so that radialLinear does not need to access the private variables, but then we'd have somewhat duplicate variables I suppose